### PR TITLE
view_templates merge revision.

### PR DESCRIPTION
**Pull request by kmasouri**

### CODE REVIEW

File: views/status.hbs [hunk 1]
```diff
@@ -48,9 +38,9 @@
 
 <main>
```
Comment (cschipani) [inline views/status.hbs]: Main?

File: config/db.js [hunk 2]
```diff
@@ -1,16 +0,0 @@
-var mysql = require('mysql');
```
Comment (cschipani) [inline config/db.js]: Was this file just leftover and is defined somewhere else? If so, thanks for removing it and ignore this comment.

File: views/status.hbs (hunk 1 above)
Comment (kmasouri) [reply views/status.hbs]: Materialize mentions keeping your stuff between 3 tags:
<header>
<main>
<footer>

File: config/db.js (hunk 2 above)
Comment (kmasouri) [reply config/db.js]: I was trying to move db info to a different file than app.js but I never finished it. The file was just left over so I got rid of it.

File: views/status.hbs (hunk 1 above)
Comment (cschipani) [reply views/status.hbs]: Neat, don't think I've heard of main before.

Comment (kmasouri) [reply views/status.hbs]: http://materializecss.com/footer.html

**Pull request merged**